### PR TITLE
fix 'subclassOf' method for the es6 inheritors

### PR DESCRIPTION
**Pull request by churkin**

### CODE REVIEW

**Review by AlekseyMartynov — DISMISSED** [review]
LGTM

File: testing/tests/DevExpress.core/class.tests.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-var Class = require("core/class");
+import Class from 'core/class';
```
Comment (ovchinnikov) [inline testing/tests/DevExpress.core/class.tests.js]: I suggest to use double quotes instead of single because all quotes in this file are double

**Review by ovchinnikov — CHANGES_REQUESTED** [review]

File: testing/tests/DevExpress.core/class.tests.js (hunk 1 above)
Comment (ovchinnikov) [reply testing/tests/DevExpress.core/class.tests.js]: Same issue with quotes in other changed files

**Comment by churkin** [comment]
FPR

**Review by ovchinnikov — APPROVED** [review]

**Review by nightskylark — APPROVED** [review]

**Review by AlekseyMartynov — APPROVED** [review]

**Pull request merged**